js/main.js

```diff
@@ -115,6 +104,11 @@ function viewchange(event) {
   }
 }
 
-$tabList.addEventListener('click', viewchange);
-$newButton.addEventListener('click', viewchange);
-$submitButton.addEventListener('click', viewchange);
+function handleViewNav(event) {
+  var dataView = event.target.getAttribute('data-view');
```

Comment:
You should probably check if there is an actual value in this attribute, and if there isn't, `return` out of the function. As of right now, if you clicked the area next to the links in the nav list but not an individual link, your content would disappear.